airflow/www/views.py

```diff
@@ -2031,6 +2031,7 @@ def trigger(self, dag_id: str, session: Session = NEW_SESSION):
                     form_fields[k]["schema"]["custom_html_form"]
                 )
         ui_fields_defined = any("const" not in f["schema"] for f in form_fields.values())
+        show_form_if_no_params = conf.getboolean("webserver", "show_trigger_form_if_no_params")
```

Comment:
```suggestion
        show_trigger_form_if_no_params = conf.getboolean("webserver", "show_trigger_form_if_no_params")
```

Reply:
...and I thought I can save a few bytes of typing - as we are in trigger form anyway :-D don't need to repeat the term trigger :-D